Added docs related to Redis version 5.2.0 (#894)
mulesoft/docs-connectors: redis/5.2/modules/ROOT/pages/redis-connector-reference.adoc

| Connection a| * <<redis_clustered, Clustered>>
* <<redis_nonclustered, NonClustered>>
 | The connection types to provide to this configuration. | | x
| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the instance expires at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  |
|===

==== Connection Types
[[redis_clustered]]
===== Clustered

====== Parameters
[%header%autowidth.spread]
|===
| Name | Type | Description | Default Value | Required
| Endpoint identification algorithm a| Enumeration, one of:

** HTTPS
** LDAPS
** DISABLED | The endpoint identification algorithm used by clients to validate the server hostname. The default value is an empty string, which means that this functionality is disabled. Clients, including client connections created by the broker for inter-broker communication, verify that the broker hostname matches the hostname in the broker's certificate. |  DISABLED |

> | TLS Configuration a| <<Tls>> |  Defines a TLS configuration, which can be used from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured sing TLS or SSL. If HTTPS is configured as the protocol, you must configure at least the keystore in the tls:context child element of the listener-config. The default is to use HTTP. | |

Reviewer: ```suggestion
| TLS Configuration a| <<Tls>> |  Defines a TLS configuration that you can use from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol, you must configure at least the keystore in the tls:context child element of the listener-config. The default is to use HTTP. | |
```

| Min Evictable Idle Time Millis a| Number |  |  1800000 |
| Soft Min Evictable Idle Time Millis a| Number |  |  1800000 |
| Num Tests Per Eviction Run a| Number |  |  3 |
| Test On Create a| Boolean |  |  false |
| Test On Borrow a| Boolean |  |  false |
| Test On Return a| Boolean |  |  false |
| Test While Idle a| Boolean |  |  false |
| Time Between Eviction Runs Millis a| Number |  |  -1 |
| Block When Exhausted a| Boolean |  |  true |
| Jmx Enabled a| Boolean |  |  true |
| Jmx Name Prefix a| String |  |  pool |
| Jmx Name Base a| String |  |  |
|===
[[redis_nonclustered]]
===== NonClustered

====== Parameters
[%header%autowidth.spread]
|===
| Name | Type | Description | Default Value | Required
| Endpoint identification algorithm a| Enumeration, one of:

** HTTPS
** LDAPS
** DISABLED |  The endpoint identification algorithm used by clients to validate the server hostname. The default value is an empty string, which means that this functionality is disabled. Clients, including client connections created by the broker for inter-broker communication, verify that the broker hostname matches the hostname in the broker's certificate. |  DISABLED |

> | TLS Configuration a| <<Tls>> |  Defines a TLS configuration, which can be used from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol, you must configure at least the keystore in the tls:context child element of the listener-config. The default is to use HTTP.  | |

Reviewer: ```suggestion
| TLS Configuration a| <<Tls>> |  Defines a TLS configuration that you can use from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol, you must configure at least the keystore in the tls:context child element of the listener-config. The default is to use HTTP.  | |
```
